backend/account/business.py

```diff
@@ -0,0 +1,15 @@
+"""
+Account business
+
+Business logic for account related operations.
+"""
+
+from . import query
+from .models import User
+
+def get_or_create_user(payload: dict) -> User:
+    return query.get_or_create_user(payload)
+
+
+def update_user(payload: dict):
+    query.update_user(payload)
```

Comment:
Newline at end of file. Need to check pylint, it should be catching this.
```suggestion
    query.update_user(payload)

```